app/results.py

```diff
@@ -247,7 +247,7 @@ def get(self):
                         results_based_on_input = False
                         query_location = ''
```

Comment:
Can you also delete this query_location *variable* which looks no longer used? It is also referred to at line 237.